packs/feats/glass-skin.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p><strong>Prerequisites</strong> You were present at the death of the medusa Alethsia, whose vitrumantic powers were passed on to you in the wake of her destruction.</p>\n<hr />\n<p>Even if vitrumancy is a mystery to you, your blood and flesh know the secrets of the art of arcane glass. Patches of your skin to flicker into glass without warning. By focusing the effect, you can concentrate the glass into a powerful protective layer of magical obsidian. Once per day, you can cast <em>@UUID[Compendium.pf2e.spells-srd.Item.Mountain Resilience]</em> on yourself as an innate primal spell, though the spell covers you in a layer of glass instead of stone. During this innate stoneskin, you gain the Shatter Glass reaction. At 18th level, the stoneskin is heightened to 6th level instead of 4th.</p>\n<hr />\n<p><strong>Shatter Glass</strong> <span class=\"pf2-icon\">r</span></p>\n<p><strong>Trigger</strong> An adjacent creature Strikes you with a melee weapon or unarmed attack and deals damage</p>\n<p><strong>Requirements</strong> Your innate stoneskin from Glass Skin is active</p>\n<p><strong>Effect</strong> You shatter some of the glass on your skin to damage your attacker. The attacker takes @Damage[3d10[slashing]] damage and the duration of the stoneskin decreases by 1 minute. At 14th level, the damage increases to @Damage[4d10[slashing]] damage, and at 18th level, the damage increases to @Damage[5d10[slashing]] damage.</p>"
+            "value": "<p><strong>Prerequisites</strong> You were present at the death of the medusa Alethsia, whose vitrumantic powers were passed on to you in the wake of her destruction.</p>\n<hr />\n<p>Even if vitrumancy is a mystery to you, your blood and flesh know the secrets of the art of arcane glass. Patches of your skin to flicker into glass without warning. By focusing the effect, you can concentrate the glass into a powerful protective layer of magical obsidian. Once per day, you can cast <em>@UUID[Compendium.pf2e.spells-srd.Item.Mountain Resilience]</em> on yourself as an innate primal spell, though the spell covers you in a layer of glass instead of stone. During this innate <em>Mountain Resilience</em>, you gain the @UUID[Compendium.pf2e.actionspf2e.Item.Shatter Glass] reaction. At 18th level, the <em>Mountain Resilience</em> is heightened to 6th level instead of 4th.</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Prerequisites</strong> You were present at the death of the medusa Alethsia, whose vitrumantic powers were passed on to you in the wake of her destruction.</p>\n<hr />\n<p>Even if vitrumancy is a mystery to you, your blood and flesh know the secrets of the art of arcane glass. Patches of your skin to flicker into glass without warning. By focusing the effect, you can concentrate the glass into a powerful protective layer of magical obsidian. Once per day, you can cast <em>@UUID[Compendium.pf2e.spells-srd.Item.Mountain Resilience]</em> on yourself as an innate primal spell, though the spell covers you in a layer of glass instead of stone. During this innate <em>mountain resilience</em>, you gain the @UUID[Compendium.pf2e.actionspf2e.Item.Shatter Glass] reaction. At 18th level, the <em>mountain resilience</em> is heightened to 6th level instead of 4th.</p>"
```